samples/va_intel/va_intel_interop.cpp

```diff
@@ -329,24 +349,31 @@ static float run(const char* fn1, const char* fn2, bool doInterop)
     va_status = vaSyncSurface(va::display, surface_id);
     CHECK_VASTATUS(va_status, "vaSyncSurface");
 
-    dumpSurface(va::display, surface_id, fn1, doInterop);
-
     cv::Size size(CLIP_WIDTH,CLIP_HEIGHT);
+    cv::Size size2(IMAGE_WIDTH,IMAGE_HEIGHT);
     cv::UMat u;
+    cv::UMat u2;
 
     cv::va_intel::convertFromVASurface(va::display, surface_id, size, u);
-    cv::blur(u, u, cv::Size(7, 7), cv::Point(-3, -3));
-    cv::va_intel::convertToVASurface(va::display, u, surface_id, size);
+    cv::va_intel::convertFromVASurface(va::display, surface_id2, size2, u2);
```

Comment:
The call is necessary since VA surface is in NV12 format while UMat data 
is in BGR format, so we have real data conversion, not simple mapping in 
memory.

On 09/18/2015 01:05 PM, krodyush wrote:
>
> In samples/va_intel/va_intel_interop.cpp 
> <https://github.com/Itseez/opencv/pull/5363#discussion_r39840598>:
>
> >
> >      cv::va_intel::convertFromVASurface(va::display, surface_id, size, u);
> > -    cv::blur(u, u, cv::Size(7, 7), cv::Point(-3, -3));
> > -    cv::va_intel::convertToVASurface(va::display, u, surface_id, size);
> > +    cv::va_intel::convertFromVASurface(va::display, surface_id2, size2, u2);
>
> why this call is needed?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Itseez/opencv/pull/5363/files#r39840598>.
>